src/firefly/js/Firefly.js

```diff
@@ -141,9 +141,13 @@ const defFireflyOptions = {
     coverage : {
         // TODO: need to define all options with defaults here.  used in FFEntryPoint.js
     },
-    multiProducesViewer : {
+    dataProducts : {
         factoryOverride: [
-        ]
+        ],
+        clientAnalysis: {
+            // xxxx : abcDataProductsAnalyzer
+
```

Comment:
If i understand correctly, the entry point of the app has to define ALL the DataProductAnalyzer available here, correct?
So, if i have one demo1 as the one you already included and i want to include a particular analyser for SOFIA FITS to chart a 1D spectra and Spitzer 1D spectra from IPAC table to chart traces grouped by column i would have something like:
`clientAnalysis: {
  demo1 : Demo1Analyzer,
  sofiaX : Sofia1DSpectra,
  spitzerA : SpectraSpitzerAnalyser
}`

Reply:
This allows a way to do the same thing from the client. It is mostly a hook now but I plan to do more with it.